docs/concept/#894-Efficient-orchestration-of-edc-negotiations/#894-Efficient-orchestration-of-edc-negotiations.md

```diff
@@ -0,0 +1,103 @@
+# #894 Efficient orchestration of edc negotiations
+
+| Key           | Value                                                                    |
+|---------------|--------------------------------------------------------------------------|
+| Creation date | 09.10.2024                                                               |
+| Ticket Id     | https://github.com/eclipse-tractusx/item-relationship-service/issues/894 |    
+| State         | DONE                                                                     | 
+
+## Table of Contents
+
+1. [Decision](#Decision)
+2. [Rationale](#Rationale)
+3. [Approach](#Approach)
+4. [Testing Strategy](#Testing-Strategy)
+
+## Decision
+
+Implementation of a new EDC Orchestrator within the EDC Client Library for efficient management of negotiations. The
+goal is to prevent duplicate negotiations for assets, thereby ensuring more efficient processing in asset consumption.
```

Comment:
The goal is to avoid redundant and unnecessary negotiations, ensuring more efficient asset utilization.
"duplicate" is my opinion  not precise enough